nexus/types/src/external_api/params.rs

```diff
@@ -1755,18 +1829,71 @@ pub struct Address {
     pub vlan_id: Option<u16>,
 }
 
+/// An address to be added or removed from an interface
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct AddressAddRemove {
+    /// The name of the interface
+    pub interface: Name,
+
+    /// The address lot this address is drawn from.
+    pub address_lot: NameOrId,
+
+    /// The address and prefix length of this address.
```

Comment:
This caught my eye since I did not see a doc string that detailed "prefix length" before.
Also slightly confusing because it is self-referential (address of this address).